TACC/protx
# Task/cooks 18   linting

## Overview: ##

Fix linting errors and refactor chart into functional component

## Related Jira tickets: ##

* [COOKS-18](https://jira.tacc.utexas.edu/browse/COOKS-18)

## Summary of Changes: ##
* fix linting errors
* Refactor chart into functional component

## Testing Steps: ##
1. run lint checks (`npm run lint`)
2. also confirm protx map works as it did before
